src/main/java/io/codelex/javaAdvancedTest/Exercise4/Reader.java

```diff
@@ -0,0 +1,22 @@
+package io.codelex.javaAdvancedTest.Exercise4;
+
+import java.io.*;
+
+public class Reader {
+
+    private static final String PATH = "C:\\Users\\ILGA\\exercises-codelex\\src\\main\\java\\io\\codelex\\javaAdvancedTest\\Exercise4/";
+
+    public static void main(String[] args) throws IOException {
```

Comment:
👍 ļoti labi, šis gan faila nosaukumu neapgriež otrādi, bet tas ir sīkums, vari atstāt kā ir

Reply:
13 rindiņā ar StringBuilder(file).reverse() man apriež arī nosaukumu otrādi, vai tad ne?

Reply:
ā jā pareizi, atvaino - nepamanīju!